tests/test_base_ophyd.py

```diff
@@ -0,0 +1,56 @@
+from __future__ import annotations
+
+import shutil
+import time as ttime
+import uuid
+from pathlib import Path
+
+import h5py
+import pytest
+
+from srx_caproto_iocs.utils import now
+
+
+@pytest.mark.cloud_friendly()
+@pytest.mark.parametrize(
+    "date_template", ["%Y/%m/", "%Y/%m/%d", "mydir/%Y/%m/%d", "disguised_spaces_%c"]
+)
+def test_base_ophyd_templates(
+    base_caproto_ioc, base_ophyd_device, date_template, num_frames=50, remove=False
+):
+    tmpdirname = f"/tmp/srx-caproto-iocs/{str(uuid.uuid4())[:2]}"
+    date = now(as_object=True)
+    write_dir_root = Path(tmpdirname)
+    dir_template = f"{write_dir_root}/{date_template}"
+    write_dir = Path(date.strftime(dir_template))
```

Comment:
I am not sure if we need to "manually" `strftime` here.  Shouldn't that happen when we stage `base_ophyd_device`?

Reply:
It's to make the directory structure on the next line. What we pass to the ophyd object and then to the caproto IOC is `dir_template`.